superset/db_engine_specs/base.py

```diff
@@ -967,25 +980,27 @@ def make_label_compatible(cls, label: str) -> Union[str, quoted_name]:
         return label_mutated
 
     @classmethod
-    def get_sqla_column_type(cls, type_: Optional[str]) -> Optional[TypeEngine]:
+    def get_sqla_column_type(
+        cls, column_type: Optional[str]
+    ) -> Tuple[Union[TypeEngine, utils.GenericDataType, None]]:
```

Comment:
I would consider just leaving the signature of this method unchanged for now; otherwise I'd just remove this one and start using the new `get_column_spec` method and implement that wherever `get_sqla_column_type` is being used, as that's the end state we want to aim for.